sdk/servicebus/service-bus/test/atomXml.spec.ts

```diff
@@ -628,36 +628,59 @@ class MockSerializer implements AtomXmlSerializer {
 });
 
 describe(`Parse empty response for list() requests to return as empty array #RunInBrowser`, function(): void {
-  mockServiceBusAtomManagementClient.sendRequest = async () => {
-    return {
-      request: new WebResource(),
-      bodyAsText: '<feed xmlns="http://www.w3.org/2005/Atom"></feed>',
-      status: 200,
-      headers: new HttpHeaders({})
-    };
-  };
-
   function assertEmptyArray(result: any) {
     assert.equal(Array.isArray(result), true, "Result must be an array");
     assert.equal(result.length, 0, "Result must be an empty array");
   }
 
   it(`List on empty list of queues gives an empty array`, async () => {
+    mockServiceBusAtomManagementClient.sendRequest = async () => {
+      return {
+        request: new WebResource(),
+        bodyAsText: '<feed xmlns="http://www.w3.org/2005/Atom"></feed>',
+        status: 200,
+        headers: new HttpHeaders({})
+      };
+    };
     const result = await mockServiceBusAtomManagementClient.listQueues();
     assertEmptyArray(result);
   });
 
   it(`List on empty list of topics gives an empty array`, async () => {
+    mockServiceBusAtomManagementClient.sendRequest = async () => {
+      return {
+        request: new WebResource(),
+        bodyAsText: '<feed xmlns="http://www.w3.org/2005/Atom"></feed>',
+        status: 200,
+        headers: new HttpHeaders({})
+      };
+    };
```

Comment:
This seems to be a copy/paste in a couple of places. Can it be refactored into a separate function? Or is there anything preventing that?

Reply:
Updated